Test fail with
> torch._inductor.exc.InductorError: LoweringException: AssertionError:
> Unregistered range symbol: yindex
>  target: aten.addmm.default

See https://github.com/pytorch/pytorch/pull/187371

author: Alexander Grund (TU Dresden)
diff --git a/test/inductor/test_max_autotune.py b/test/inductor/test_max_autotune.py
--- a/test/inductor/test_max_autotune.py
+++ b/test/inductor/test_max_autotune.py
@@ -913,6 +913,7 @@ class TestMaxAutotune(TestCase):
         "Hopper-style mm_persistent_tma template is shadowed by the Blackwell warp-specialized TMA template on data-center Blackwell; covered by test_max_autotune_blackwell.py",
     )
     @skipIfXpu(msg="XPU TMA requires contiguous last dimension")
+    @unittest.skip("187371")
     @parametrize("a_transposed", (False, True))
     @parametrize("b_transposed", (False, True))
     @parametrize("dynamic", (False, True))
